src/hotspot/share/compiler/compileTask.cpp

```diff
@@ -22,7 +22,6 @@
  *
  */
 
-#include "code/SCCache.hpp"
```

Comment:
This compilation unit uses `AOTCode*` stuff, so the include for `aotCodeCache.hpp` should be here?

Reply:
The only reference is `AOTCodeEntry*`  which is declared in `compileTask.hpp`. No methods from it are called.

Reply:
Right, OK.